index.html

```diff
@@ -0,0 +1,399 @@
+                                    <option>
+                                        Patnering with us as
+                                    </option>
+                                    <option>
+                                        2
+                                    </option>
+                                    <option>
+                                        3
+                                    </option>
+                                </select>
+                            </span>
+                        </div>
+                    </div>
+                    <div class="">
+                        <div class="join_the_team">
+                            <h4>
+                                <b>
+                                    Details Overseas Stay
+                                </b>
+                            </h4>
+                        </div>
+                        <div class="">
+                            <span>
+                                <input name="name" type="text" value=""/>
+                            </span>
+                            <span>
+                                <select class="dropdown_select overseas_select">
+                                    <option>
+                                        1
+                                    </option>
+                                    <option>
+                                        2
+                                    </option>
+                                    <option>
+                                        3
+                                    </option>
+                                </select>
+                            </span>
+                            <span>
+                                <select class="dropdown_select overseas_select">
+                                    <option>
+                                        1
+                                    </option>
+                                    <option>
+                                        2
+                                    </option>
+                                    <option>
+                                        3
+                                    </option>
+                                </select>
+                            </span>
+                        </div>
+                        <div class="">
+                            <textarea cols="80" name="name" rows="4" class="details_oversea_textarea">
+                            </textarea>
+                        </div>
+                    </div>
+                    <div class="submit_button">
+                        <button name="button" type="submit">
+                            <b>
+                                SUBMIT
+                            </b>
+                        </button>
+                    </div>
+                </form>
+
+            </section>
+    
+        </main>
+    <footer>
+    <div id="footer">
```

Comment:
The div with id footer seems unwanted. The content can be included in footer tag.